recipes-samples/images/lmp-gateway-image.bb

```diff
@@ -24,45 +38,20 @@ CORE_IMAGE_BASE_INSTALL += " \
 "
 
 CORE_IMAGE_BASE_INSTALL += " \
-    docker \
-    bluez5-noinst-tools \
     ethtool \
     git \
-    jool \
     less \
-    linux-firmware-ar3k \
-    linux-firmware-ath9k \
-    linux-firmware-ath10k \
-    linux-firmware-qca \
-    linux-firmware-wl18xx \
     openssh-sftp-server \
     packagegroup-core-full-cmdline-utils \
     packagegroup-core-full-cmdline-extended \
     packagegroup-core-full-cmdline-multiuser \
     python3-compression \
     python3-distutils \
-    python3-docker \
-    python3-docker-compose \
     python3-json \
     python3-multiprocessing \
     python3-netclient \
     python3-pkgutil \
     python3-shell \
     python3-unixadmin \
```

Comment:
I was curious if we could categorize these python3 modules?

Reply:
Most of them are there because of ansible (iirc sftp-server as well), so we might want to split those to a different file at least.